doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,97 @@
 # Python SDK
 
-More docs coming soon!
+The code for the Python SDK lives in `/econia/src/python/sdk/econia_sdk` ([link](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)). There are 2 primary packages (`econia_sdk.entry` and `econia_sdk.view`) complemented by two secondary imports
+
+# Primary Packages
+## `econia_sdk.entry`
+
+This package contains helpers for the `public entry` (that is, "targetable by transactions") functions possessed by the Econia exchange. Each method's name corresponds to the name of a public entry function in one of the following Econia Move modules:
```

Comment:
Will you please simplify the first sentence to:

"This package contains helpers for Econia's `public entry` functions."